bot/kodiak/queries.py

```diff
@@ -1070,7 +1070,7 @@ async def create_notification(
             name=CHECK_RUN_NAME,
             head_sha=head_sha,
             status="completed",
-            completed_at=datetime.utcnow().isoformat(),
+            completed_at=datetime.utcnow().replace(tzinfo=timezone.utc).isoformat(),
```

Comment:
```suggestion
            completed_at=datetime.now(timezone.utc).isoformat(),
```

Reply:
Seems that `.utcnow()` doesn't set the timezone on the date time, instead it makes it a naive datetime without a timezone -- weird